R/OMLData.R

```diff
@@ -201,9 +200,25 @@ OMLData = R6Class("OMLData",
       }
       if (self$parquet) {
         path = self$parquet_path
-        private$.backend = mlr3db::as_duckdb_backend(path)
+        backend = mlr3db::as_duckdb_backend(path)
+        if (!test_names(backend$colnames, type = "strict")) {
+          new = make.names(backend$colnames)
+          if (length(unique(new)) != length(new)) {
```

Comment:
you can tell `make.names()` to create unique names

Reply:
good to know. But in this case I don't think we should use it because the names an appear in different objects and I would not assume that is guaranteed that the names appear everywhere in the same order.